File: C/1038 - Lanche.c
/* 
   ==UserScript==
 @name         Lanche
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1038 - Lanche
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

#include <stdio.h>
#include <math.h>

// DAVID NUNES RIBEIRO - INF 1A

int main()
{
    int Codigo, Quantidade;
    double Total;
    
    scanf("%d%d", &Codigo, &Quantidade);
    
    if(Codigo == 1){
        Total = 4.00 * Quantidade;
    }
    else if(Codigo == 2){
        Total = 4.50 * Quantidade;
    }
    else if(Codigo == 3){
        Total = 5.00 * Quantidade;
    }
    else if(Codigo == 4){
        Total = 2.00 * Quantidade;
    }
    else if(Codigo == 5){
        Total = 1.50 * Quantidade;
    }
    
    printf("Total: R$ %.2lf\n", Total);

    return 0;
}

File: Java/1038 - Lanche.java
/* 
   ==UserScript==
 @name         Lanche
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1038 - Lanche
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/


import java.util.Scanner;
import java.io.IOException;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        // Scanner
        Scanner scan = new Scanner(System.in);
        
        int codItem = scan.nextInt();
        int qntItem = scan.nextInt();
        double total;
        
        if(codItem == 1){
            total = 4.00 * qntItem;
            System.out.println(String.format("Total: R$ %.2f", total));
        }
        else if(codItem == 2){
            total = 4.50 * qntItem;
            System.out.println(String.format("Total: R$ %.2f", total));
        }
        else if(codItem == 3){
            total = 5.00 * qntItem;
            System.out.println(String.format("Total: R$ %.2f", total));
        }
        else if(codItem == 4){
            total = 2.00 * qntItem;
            System.out.println(String.format("Total: R$ %.2f", total));
        }
        else{
            total = 1.50 * qntItem;
            System.out.println(String.format("Total: R$ %.2f", total));
        }
    }
}
